Android/app/src/main/java/com/vigbag/android/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,72 @@
+package com.vigbag.android.ui.home
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.vigbag.android.ui.home.adapters.ParentAdapter
+import com.vigbag.android.model.ParentItemDataClass
+import com.vigbag.android.R
+import com.vigbag.android.databinding.FragmentHomeBinding
+import com.vigbag.android.model.ChildItemDataClass
+
+class HomeFragment : Fragment(R.layout.fragment_home) {
+
+    private var _binding: FragmentHomeBinding? = null
+    private val binding get() = _binding!!
+
+    private lateinit var parentAdapter: ParentAdapter
+    private lateinit var parentItemList:ArrayList<ParentItemDataClass>
+
+    private lateinit var headphoneList:ArrayList<ChildItemDataClass>
+    private lateinit var tshirtList:ArrayList<ChildItemDataClass>
+    private lateinit var phoneList:ArrayList<ChildItemDataClass>
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        _binding=FragmentHomeBinding.bind(view)
+        onaddAllData()
+    }
+
+    private fun onaddAllData() {
+        headphoneList=ArrayList()
```

Comment:
Every assignment should have appropriate spacing.
For example, here it should be `headphoneList = ArrayList()`. Spacing is very important in order to make your code look nice. Please change this everywhere because I can see it at a lot of places. After that I think we should be good to go ahead and merge your work.

Reply:
@07jasjeet  Sure I will add Spacing in assignment